src/SlackConnector/SlackConnection.cs

```diff
@@ -160,15 +160,27 @@ public void Disconnect()
             }
         }
 
-        public async Task Say(BotMessage message)
+        public async Task<SlackMessagePosted> Say(BotMessage message)
         {
```

Comment:
In my PR I amended your existing SayTests to take the result and compare the result returned from the server. If that's not sufficient, can you give me some more details about what you're expecting?